test/Common.FunctionalTests/Inprocess/LoggingTests.cs

```diff
@@ -91,5 +91,35 @@ public class LoggingTests : IISFunctionalTestBase
             }
         }
 
+        [ConditionalFact]
+        public async Task StartupMessagesLogFileSwitchedWhenLogFilePresentInWebConfig()
```

Comment:
Did we fix a bug where managed stdout wasn't available in test output?

Reply:
This PR doesn't address that. The other one does.